stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_file_attachments.xml

```diff
@@ -44,26 +44,44 @@
         tools:src="@tools:sample/avatars"
         />
 
-    <ImageView
-        android:id="@+id/deliveryStatusIcon"
-        android:layout_width="16dp"
-        android:layout_height="16dp"
-        android:layout_marginEnd="4dp"
-        app:layout_constraintBottom_toBottomOf="@+id/tvTime"
-        app:layout_constraintEnd_toStartOf="@+id/tvTime"
-        app:layout_constraintTop_toTopOf="@+id/tvTime"
-        tools:src="@drawable/stream_ui_ic_check_all"
-        />
-
-    <TextView
-        android:id="@+id/tvTime"
+    <androidx.appcompat.widget.LinearLayoutCompat
```

Comment:
Seems like a lot of doubling. What if we extract it to a standalone layout file and use it through the `include` mechanism? Or even we could create a custom view and put this to the required places.

Reply:
IMO, include will work fine for now.